models/youtube__video_metadata.sql

```diff
@@ -1,20 +1,15 @@
-{{ config(enabled=var('youtube__using_video_metadata', false)) }}
-
 with video_metadata as (
     select *
-    from {{ var('video_metadata')}}
+    from {{ var('video') }}
 
 ), url_parsing as (
 
     select
-        video_id,
-        title,
-        description,
-        published_at,
-        channel_title,
-        {{ fivetran_utils.json_parse("thumbnails",["default","url"]) }} as default_thumbnail_url,
-        {{ fivetran_utils.json_parse("thumbnails",["medium","url"]) }} as medium_thumbnail_url,
-        {{ fivetran_utils.json_parse("thumbnails",["high","url"]) }} as high_thumbnail_url
+        *,
```

Comment:
Yeah there are soooo many new fields being added that were not in the original video_metadata table. Therefore, I felt it would be more appropriate to just do a select * here.